lib/services/registrasquadra_db.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:alley_app/model/data.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class RegistraSquadraDbService {
+  final CollectionReference usersCollection = Firestore.instance.collection('users');
+  final CollectionReference teamsCollection = Firestore.instance.collection('teams');
+
+  Future<bool> registerTeam(String idSquadra, String regione, String nome, String girone, Sesso sesso,
+      String campionato, String coachId) async {
+    if (await this.teamAlreadySigned(idSquadra)) {
+      return false;
+    } else {
+      await usersCollection.document(coachId).updateData({'idSquadra': idSquadra});
+      await teamsCollection.document().setData({
+        'idSquadra': idSquadra,
+        'regione': regione,
+        'nome': nome,
+        'girone': girone,
+        'sesso': sesso.toString().split('.')[1],
```

Comment:
enums should be saved in the database with their `toString` representation for extensibility purposes when retrieving the value.

 Sex in our context should always be binary though, so your choice